balancer-labs/frontend-v2
# TRM Sanctioned Wallets

# Description

Checks if a wallet is sanctioned against the TRM labs sanctions API. If sanctioned the wallet connector will disconnect the wallet immediately while also prompting the user as to why.

## Type of change


## How should this be tested?

Connect with non sanctioned wallet, should be able to use app.
Connect with sanctioned wallet - should not be able to transact with app.

## Visual context

N/A

## Checklist:
